_modules/qubes_dom0_update.py

```diff
@@ -3311,3 +3319,109 @@ def list_installed_patches():
         salt '*' pkg.list_installed_patches
     '''
     return _get_patches(installed_only=True)
+
+def mod_init(low):
```

Comment:
According to this very comment copied from `salt.states.pkg`, it should be named `pkg.ex_mod_init`.